### ISSUE TITLE
RARBG API is not responding or returning errors!

### ISSUE THREAD

**Issue by tobeychris** [psouza4/mediacentermaster#1882]
Issue confirmed by multiple people on forum:
http://forums.mediacentermaster.com/viewtopic.php?f=4&t=20293
```
9:24 PM :: RARBG.to:  Searching for "cool show S04E07"
9:24 PM :: [21:24:48.902] Fetching URL (GET):  https://torrentapi.org/pubapi_v2.php?get_token=get_token&app_id=Media+Center+Master
9:24 PM :: RARBG.to:  Invalid response from RARBG:
9:24 PM :: MCM:  RARBG API is not responding or returning errors!
```
If I visit this site myself, it returns a JSON response with token.
`{"token":"abcdefg"}`

**Comment by AlexMarcoDAngelo** [comment]
This has been going on for a quite a bit now :(

**Comment by tobeychris** [comment]
Fixed in Version 2.21, build 11921.848!
